lms/static/js/header_factory.js

```diff
@@ -0,0 +1,16 @@
+;(function (define) {
```

Comment:
Nit: I think we usually name JS files using hyphens rather than underscores to separate names.